commcare_connect/opportunity/forms.py

```diff
@@ -303,3 +303,36 @@ def __init__(self, *args, **kwargs):
             self.fields["deliver_units"].initial = [
                 deliver_unit.pk for deliver_unit in self.instance.deliver_units.all()
             ]
+
+
+class SendMessageMobileUsersForm(forms.Form):
+    title = forms.CharField(
+        empty_value="Notification from CommCare Connect",
+        required=False,
+        widget=forms.TextInput(attrs={"placeholder": "Notification from CommCare Connect"}),
+    )
+    body = forms.CharField(widget=forms.Textarea)
+    message_type = forms.MultipleChoiceField(
+        choices=[("notification", "Push Notification"), ("sms", "SMS")],
+        widget=forms.CheckboxSelectMultiple,
+    )
+
+    def __init__(self, *args, **kwargs):
+        users = kwargs.pop("users", [])
+        super().__init__(*args, **kwargs)
+
+        self.helper = FormHelper(self)
+        self.helper.layout = Layout(
+            Row(Field("selected_users")),
+            Row(Field("title")),
+            Row(Field("body")),
+            Row(Field("message_type")),
+            Submit(name="submit", value="Submit"),
+        )
+
+        choices = [(user.pk, f"{user.name} - {user.username}") for user in users]
+        self.fields["selected_users"] = forms.MultipleChoiceField(
+            choices=choices,
+            help_text="Use Ctrl-click to select multiple users",
```

Comment:
I don't think this is necessary anymore

Reply:
Fixed. https://github.com/dimagi/commcare-connect/pull/240/commits/6a97662b8cb04970ec2855fc71861bb6d491d60f